code/modules/codex/entries/paperwork.dm

```diff
@@ -22,7 +22,8 @@
 \[hr\] : Adds a horizontal rule.<br>
 \[list\] - \[/list\] : A list. (Does not work with crayons)<br>
 \[*\] : A dot used for lists. (Does not work with crayons)<br>
-\[logo\] - Inserts EXO logo image.<br>
+\[logo\] - Inserts CEC logo image.<br>
```

Comment:
these macros are a bit confusing, they're both the cec logo. maybe change the second one to be more descriptive of context

Which one is normal and which is formal

Reply:
Fixed in #be1b184